express_relay/sdk/js/src/examples/simpleSearcherSvm.ts

```diff
@@ -102,15 +102,20 @@ class SimpleSearcherSvm {
     ixDummy.programId = dummyPid;
 
     const txRaw = new anchor.web3.Transaction().add(ixDummy);
-
+    const expressRelayConfig = await this.client.getExpressRelaySvmConfig(
```

Comment:
Maybe it's a better idea to not fetch config here. We can store it on client as a singleton?

Reply:
not certain how that would look like, we can discuss this.